languages/python/exercises/concept/bools/.docs/after.md

```diff
@@ -0,0 +1,58 @@
+Python represents True and False values with the [boolean][boolean] type. There are only two [boolean][boolean] values: _True_ and _False_. These values can be assigned to a variable and combined with [boolean operators][boolean-operators] (`and`, `or`, `not`):
+
+```python
+true_variable = True and True
+false_variable = True and False
+
+true_variable = False or True
+false_variable = False or False
+
+true_variable = not False
+false_variable = not True
+```
+
+[Boolean operators][boolean-operators] use _short-circuit evaluation_, which means that expression on the right-hand side of the operator is only evaluated if needed.
+
+Each of the operators has a different precedence, where `not` is evaluated before `and` and `or`. Brackets can be used to evaluate one part of the expression before the others:
+
+```python
+not True and True # => False
+not (True and False) # => True
+```
+
+## Type Coercion
+
+The [`bool` function][bool-function] converts any type to a Boolean value. By default all values return `True` unless defined to return `False`.
+
+Some built-ins already defined to be considered `False`:
+
+- constants like `None` and `False`
+- zero of any _numeric type_
+- empty _sequences_ and _collections_
+
+```python
+bool(1) # => True
+bool(0) # => False
```

Comment:
I thought we were going to avoid the topic of truthiness (for simplicity)? If we're casting values into `bool`, that warrants a truthiness mention.

Reply:
I didn't want to discuss it in the exercise itself, but thought it could work in the after document to further expand on it.

Since type coercion is so similar to truthiness I can add a sentence to bring that into the description

Reply:
See sentences on line 38-47